packages/suite/src/constants/suite/inputs.ts

```diff
@@ -1,17 +1,17 @@
-export enum MAX_LENGTH {
-    PIN = 50,
-    PASSPHRASE = 50,
+export const MAX_LENGTH = {
```

Comment:
Why this chance?

Reply:
Enums are usually used to create a collection of mutually exclusive constants, where each name maps to a unique value. I changed it because of this https://typescript-eslint.io/rules/no-duplicate-enum-values/ rule which makes sense to me